vtk_viewer/CMakeLists.txt

```diff
@@ -14,9 +14,10 @@ find_package(VTK REQUIRED NO_MODULE)
 if(VTK_FOUND AND ("${VTK_VERSION}" VERSION_LESS 7.1))
   message(FATAL_ERROR "The minimum required version of VTK is 7.1, but found ${VTK_VERSION}")
   set(VTK_FOUND FALSE)
-else()
-  include(${VTK_USE_FILE})
 endif()
+#else()
+#  include(${VTK_USE_FILE})
+#endif()
```

Comment:
Is this not needed?

Reply:
I'm really not sure if it is needed on bionic or not, but it causes problems on focal. If I had that, I got undefined reference errors from VTK functions. If this turns out to be a hugely breaking PR, I would vote for tagging the prior commit and saying new features will be available on Noetic. Like I said in the PR, I think this whole repo needs a CMake overhaul.